_posts/2021/2021-02-04-cypress.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Cypress - our tool of choice for Quality Assurance
+date: 2021-02-04
+header_image: public/quality-assurance.jpg
```

Comment:
This image is a bit generic and thus similar to some we already have. Maybe we could go a bit deeper into the QA direction? 

Here are some examples:
- https://pixabay.com/illustrations/software-testing-service-762486/
- https://pixabay.com/illustrations/to-do-list-business-checklist-form-4483048/